docs/ops/sort/MulticlassNMS_8.md

```diff
@@ -0,0 +1,147 @@
+3.  If `B` is empty then go to step 9.
+4.  Take the box with highest score. Suppose that it is the box `b` with the score `s`.
+5.  Delete `b` from `B`.
+6.  If the score `s` is greater or equal than `score_threshold`  then add `b` to `D` else go to step 9.
+7.  If `nms_eta < 1` and `adaptive_threshold > 0.5`, update `adaptive_threshold *= nms_eta`.
+8.  For each input box `b_i` from `B` and the corresponding score `s_i`, set `s_i = 0` when `iou(b, b_i) > adaptive_threshold`, and go to step 3.
+9.  Return `D`, a collection of the corresponding scores `S`, and the number of elements in `D`.
+
+This algorithm is applied independently to each class of each batch element. The total number of output boxes for each class must not exceed `nms_top_k`. Boxes of `background_class` are skipped and thus eliminated. After NMS step, it selects at most `keep_top_k` scoring candidate boxes per batch element.
+
+**Attributes**:
+
+* *sort_result_byclass*
+
+  * **Description**: *sort_result_byclass* is a flag that specifies whether to sort selected boxes by class or not.
+  * **Range of values**: true of false
+    * *true* - sort selected boxes by class id (ascending) in each batch element.
+    * *false* - do not sort selected boxes by class in each batch element, by score instead.
+  * **Type**: boolean
+  * **Default value**: false
+  * **Required**: *no*
+
+* *output_type*
+
+  * **Description**: the output tensor type
+  * **Range of values**: "i64" or "i32"
+  * **Type**: string
+  * **Default value**: "i64"
+  * **Required**: *No*
+
+**Inputs**:
+
+*   **1**: `boxes` - tensor of type *T* and shape `[num_batches, num_boxes, 4]` with box coordinates. he box cooridnates are layout as `[xmin, ymin, xmax, ymax]`. Required.
+
+*   **2**: `scores` - tensor of type *T* and shape `[num_batches, num_classes, num_boxes]` with box scores. Required.
+
+*   **3**: `nms_top_k` - scalar or 1D tensor with 1 element of type *T_MAX_BOXES* specifying maximum number of boxes to be selected per class. Optional with default value 0 meaning select no boxes.
+
+*   **4**: `iou_threshold` - scalar or 1D tensor with 1 element of type *T_THRESHOLDS* specifying intersection over union threshold. Optional with default value 0 meaning keep all boxes.
+
+*   **5**: `score_threshold` - scalar or 1D tensor with 1 element of type *T_THRESHOLDS* specifying minimum score to consider box for the processing. Optional with default value 0.
+
+*   **6**:  `background_class` - scalar or 1D tensor with 1 element of type *T_IND* specifying the background class index. Optional with default value -1 meaning keep all classes.
+
+*   **7**:  `keep_top_k` - scalar or 1D tensor with 1 element of type *T_MAX_BOXES* specifying maximum number of boxes to be selected per batch element. Optional with default value -1 meaning keep all boxes.
+
+*   **8**:  `nms_eta` - scalar or 1D tensor with 1 element of type *T_THRESHOLDS* specifying eta parameter for adpative NMS. Optional with default value 1.0.
```

Comment:
Consider changing some inputs to be attributes.

Reply:
This kind of NMS so far only find existing in framework PDPD. And all these are attributes. So I moved all them there.